samples/dashboards/demo/cloud-monitoring/demo.js

```diff
@@ -0,0 +1,724 @@
+                            }
+                        }
+                    }, {
+                        id: 'ami',
+                        responsive: {
+                            small: {
+                                width: '100%'
+                            },
+                            medium: {
+                                width: '50%'
+                            },
+                            large: {
+                                width: '25%'
+                            }
+                        }
+                    }, {
+                        id: 'os',
+                        responsive: {
+                            small: {
+                                width: '100%'
+                            },
+                            medium: {
+                                width: '50%'
+                            },
+                            large: {
+                                width: '25%'
+                            }
+                        }
+                    }]
+                }, {
+                    cells: [{
+                        height: 400,
+                        layout: {
+                            rows: [{
+                                cells: [{
+                                    id: 'cpu',
+                                    height: 200
+                                }, {
+                                    id: 'memory',
+                                    height: 200
+                                }]
+                            }, {
+                                cells: [{
+                                    id: 'health',
+                                    height: 195
+                                }, {
+                                    id: 'disk',
+                                    height: 195
+                                }]
+                            }]
+                        },
+                        responsive: {
+                            small: {
+                                width: '100%'
+                            },
+                            medium: {
+                                width: '100%'
+                            },
+                            large: {
+                                width: '50%'
+                            }
+                        }
+                    }, {
+                        id: 'disk-usage',
+                        height: 400,
+                        responsive: {
+                            small: {
+                                width: '100%'
+                            },
+                            medium: {
+                                width: '100%'
+                            },
+                            large: {
+                                width: '50%'
+                            }
+                        }
+                    }]
+                }, {
+                    cells: [{
+                        id: 'instances-table',
```

Comment:
The _Instances_ data grid is apparently 400px high in mobile view, causing a large padding below even when there are no other cells on the same row. Do we have a simple fix for that now? Or wait until v2 maybe?

Reply:
Until v2, let's leave it like it is. In the next version, we can use media queries.